networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -233,12 +236,22 @@ def extend_hosting_port_info(self, context, port_db, hosting_device,
             hosting_info['cidr_exposed'] = ext_dict['cidr_exposed']
             hosting_info['gateway_ip'] = ext_dict['gateway_ip']
         else:
+            router_id = port_db.device_id
+            router = self.l3_plugin.get_router(context, router_id)
+            # skip routers not created by the user -- they will have
+            # empty-string tenant IDs
+            if router.get(ROUTER_ROLE_ATTR):
+                return
+            snat_ips = self.apic_driver.get_snat_ip_for_vrf(context,
+                router['tenant_id'], net)
```

Comment:
discussed with Mandeep. He is okay to merge this per tenant SNAT ip approach for now so we have a solution. But he wants us to open a support case (enhancement + high priority + congo) to track the "per vrf" and common snat ip case.